themes/custom/droopler_theme/scss/components/_d_p_side_tiles.scss

```diff
@@ -92,15 +92,43 @@ $d-p-side-tiles-half-padding: 10% !default; // Spacing between content and half
     .field--name-field-d-media-image {
       background-color: $d-p-side-tiles-image-background-color;
 
-      a {
-        display: block;
-      }
       img {
         width: 100%;
         height: auto;
         margin: 0 !important;
         display: block;
       }
+
+      a {
+        display: flex;
+        justify-content: center;
+        align-items: center;
+
+        &[href*=".svg"] {
+          position: relative;
+
+          &::before {
+            content: '';
+            display: block;
+            padding-bottom: calc(100% / (466 / 307));
+          }
+
+          .no-image-style {
```

Comment:
Masz dwa razy no-image-style z bardzo podobnym stylem (różnica tylko w em) może powinna to być jakas osobna klasa tak wystylowana przekazana w templatce, albo przynajmniej extendowana?